cupyx/jit/_interface.py

```diff
@@ -169,6 +169,11 @@ def cached_code(self):
 
 def rawkernel(*, mode='cuda', device=False):
     """A decorator compiles a Python function into CUDA kernel.
+    Args:
+        mode ('numpy' or 'cuda'): The rule for typecast.
+        device (int or cupy.cuda.Device): Index of the device to be used.
+        Be careful that the device ID (a.k.a. GPU ID) is zero origin.
+        The current device is selected by default.
```

Comment:
Content aside, this formatting is incorrect. The 2nd and further lines should be indented.
```suggestion
        device (int or cupy.cuda.Device): Index of the device to be used.
            Be careful that the device ID (a.k.a. GPU ID) is zero origin.
            The current device is selected by default.
```
During PR review, a preview of the doc updates can be checked in the CI, e.g. https://cupy--8857.org.readthedocs.build/en/8857/reference/generated/cupyx.jit.rawkernel.html. I suggest to build the docs locally to confirm the changes, before pushing to a PR branch.

Reply:
`device` here does not mean that this decorator takes a `Device` object or a device ID, but rather indicating this is a device function, not a kernel.